scripts/signature.py

```diff
@@ -93,7 +97,50 @@ def diff(args):
         """
         )
 
-    return exit_status
+    return delta, exit_status
+
+
+def diff(args):
+    """Diffs Velox function signatures."""
+    return diff_signatures(args)[1]
+
+
+def bias(args):
```

Comment:
this logic is not all that trivial anymore, can we have unit tests for this function?

Reply:
Added tests..